subsys/bluetooth/audio/has.c

```diff
@@ -945,6 +988,29 @@ static ssize_t write_control_point(struct bt_conn *conn, const struct bt_gatt_at
 	return len;
 }
 
+#if defined(CONFIG_BT_HAS_FEATURES_NOTIFIABLE)
+static void features_work_process(struct k_work *work)
+{
+	for (size_t i = 0U; i < ARRAY_SIZE(has_client_list); i++) {
+		struct has_client *client = &has_client_list[i];
+		int err;
+
+		if (client->conn == NULL) {
+			/* mark to notify on reconnect */
+			client->pending_ntf.active_index = true;
+			continue;
+		}
+
+		err = bt_gatt_notify(client->conn, FEATURES_ATTR,
+				     &has.features, sizeof(has.features));
+		if (err != 0) {
+			LOG_DBG("failed to notify features for %p: %d",
+				client->conn, err);
```

Comment:
```suggestion
			LOG_DBG("failed to notify features for %p: %d", client->conn, err);
```

The maximum line length have been increased to 100 :) Please use that going forwards